sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -21,6 +21,29 @@ directive:
     $["x-accessibility"] = "internal"
 ```
 
+### Make the API version parameterized so we generate a multi-versioned API
+
+This should be fixed in the swagger, but we're working around it locally for now.
```

Comment:
what is the issue?

Reply:
adding an API version as a parameter to the endpoint instead of having it fixed.